sidekick_core/lib/sidekick_core.dart

```diff
@@ -382,3 +382,15 @@ class OutOfCommandRunnerScopeException implements Exception {
     return "OutOfCommandRunnerScopeException{message: $message}";
   }
 }
+
+extension GetPackage on ArgResults {
+  /// Returns [DartPackage] from first argument in [rest] or if [rest] is empty from [entryWorkingDirectory]
+  DartPackage get package {
```

Comment:
tbh, I don't see this as part of `sidekick_core`

Reply:
moved it to `dart_package.dart`